types/hooks/screens/useSignedChannelListScreenHook.types.ts

```diff
@@ -5,6 +5,7 @@ interface UseSignedChannelListScreenHook {
   goToChatScreen: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
   goToCommunityScreen: (channel: ChannelList) => void;
+  goToContactScreen: () => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new function `goToContactScreen` has been added to the interface but it doesn't accept any parameters. If this function is intended to navigate to a specific contact's screen, it might need an identifier parameter (like a contact ID) to know which contact's screen to navigate to.
```diff
-   goToContactScreen: () => void;
+   goToContactScreen: (contactId: string) => void;
```
If it's meant to navigate to a general contact list or similar, then it's fine as it is.


<!-- This is an auto-generated comment by OSS CodeRabbit -->